src/promptflow-devkit/promptflow/_sdk/_utilities/serve_utils.py

```diff
@@ -126,10 +131,48 @@ def serve_python_flow(
         target = f"http://{host}:{port}"
         logger.info(f"Opening browser {target}...")
         webbrowser.open(target)
-    # Debug is not supported for now as debug will rerun command, and we changed working directory.
     app.run(port=port, host=host)
 
 
+def serve_python_flow(
+    *,
+    flow_file_name,
+    flow_dir,
+    port,
+    host,
+    static_folder,
+    config,
+    environment_variables,
+    init,
+    skip_open_browser: bool,
+    async_subprocess: bool = False,
+):
```

Comment:
add some docstring on the return value?

Reply:
maybe add a new function mainly for async scenario, instead of add a bool parameter

Reply:
Sure, I have planned to do some more refactor inside serve_utils